committee-steering/governance/annual-reports.md

```diff
@@ -0,0 +1,125 @@
+reports are submitted so there is an established point of contact for any help or
+guidance needed
+3. The March SIG Chairs and Tech Leads meeting will be used as follow up and
+report retrospectives
+4. Steering will schedule private meetings to follow up on anything denoted as
+a private matter on the report; reports are not made public until all private
+comments have been addressed
+5. Once reviewed and follow ups are completed, Steering will ask the chair(s)
+of the group to submit a pull request to that groups documentation in the
+community repo.
+6. The Steering Committee liaison will work directly with groups that have follow
+up items and update Steering during regular monthly meetings
+
+### Tips for Chairs:  
+- SIG Chairs are responsible for collecting and reporting information for
+sponsored Working Groups.    
+- Respond with n/a to questions that are not applicable.  Most questions will be.
+- Please include any and all private matters between [private] tags [/private].
+The published report will not include any private matters. All reports are treated
+as private until Steering asks for a PR, which will not include private tags.
+- Wait until Steering gives you the go ahead before submitting a pull request.  
+[TODO what else?]
+
+## Questions for report:
+
+Copy these questions as a template with the applicable sections   
+
+#### You and your role:
+- Do you enjoy the role? What do you find challenging? How can we/is there
+something we can provide that would better support you? Do you want to continue
+or find a replacement?
+- Do you have goals for the group? If you feel that you aren’t ready to pass the
+baton, what would you like to accomplish before you do?
+- How can Steering help? Do you have feedback for us? Suggestions for what we
+should work on?
+
+#### Special Interest Groups:
+- How are you doing with operational tasks in [yourgroup]-governance.md?
+- Is your README accurate? have a CONTRIBUTING.md file?
+- All subprojects correctly mapped and listed in sigs.yaml?
+- Are all subproject maintained? Are there any that are being retired?
+- Tell us about your meetings: large/small, active/quiet? learnings? up to date
+on the Kubernetes community YouTube under a community group playlist? Meeting
+notes up to date?
+- When was your last public community update? (provide link)
+
+##### Membership
+- Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+- How do you measure membership? By mailing list members, OWNERs, or something
+else?
+- How does the group measure reviewer and approver bandwidth? Do you need help in
+any area now? What are you doing about it?
+- Is there a healthy onboarding and growth path for contributors in your SIG? What
+ are some activities that the group does to encourage this?What programs are you
+  participating in to grow contributors throughout the contributor ladder?
+- What programs do you participate in for new contributors?
```

Comment:
Can we ask them what support they'd like from us in this regard? Or maybe that's more contribex than steering... :thinking: